xnmt/encoder.py

```diff
@@ -159,6 +159,8 @@ def transduce(self, src):
     # convolution and pooling layers
     #l1 = (W*src)+b
     l1 = dy.affine_transform([b, W, src])
+    if self.with_bias is False:
+      l1 = W * src
     output = l1
     if self.nonlinearity is 'linear':
```

Comment:
Instead of:

    if x:
    else:
      if y:
      else:

you should use

    if x:
    elif y:
    else: